projects/packages/my-jetpack/src/class-products.php

```diff
@@ -176,23 +176,16 @@ public static function get_products() {
 	 * @return array
 	 */
 	public static function get_products_by_ownership( $type ) {
```

Comment:
Not related to this PR directly - but this is a method that we should avoid calling outside of the My Jetpack context. Even though we don't load the full product array by calling `get_products`, the `get_status` call is still going to do checks for the presence of plans by sending requests to WPCOM.

That should be fine in the context of My Jetpack where we have already loaded products once and cached the results for that request lifecycle. But if we call this elsewhere, it will likely end up making some unintended API requests.